### ISSUE TITLE
Update for Unity 5.11.1

### ISSUE THREAD

**Issue by woodne** [unitycontainer/microsoft-dependency-injection#55]
The build appears to be failing, and when I update my references to abstractions, using this package brings in the wrong version of Unity.Container, and I get errors like:

```
System.TypeLoadException: Method 'Get' in type 'Unity.Builder.BuilderContext' from assembly 'Unity.Container, Version=5.10.3.0, Culture=neutral, PublicKeyToken=489b6accfaf20ef0' does not have an implementation.
>>    at Unity.UnityContainer..ctor()
```

**Comment by pnmcosta** [comment]
Yup, same here.

I was trying to accomplish this https://stackoverflow.com/a/51036884/271433

**Comment by pnmcosta** [comment]
@woodne with these package versions:

Unity.Abstractions 4.1.2
Unity.Container 5.10.2
Unity.Microsoft.DependencyInjection 5.10.1

It seems to work but I'm still testing

**Comment by karelkral** [comment]
@woodne I have a terrible experience with these packages when I updating NuGet. Every second release does not work correctly. My working version for `netcoreapp2.2` are:

    <PackageReference Include="Unity.Container" Version="5.10.2" />
    <PackageReference Include="Unity.Microsoft.DependencyInjection" Version="5.10.1" />

But sadly, these references do not work for `netcoreapp3.0`

**Comment by Jeromino** [comment]
I've had the same experience and I hope prerelease packages will first be released next time.

The owner of the repo was planning an upgrade to 3.0 and I we should try to help as well.

**Comment by pnmcosta** [comment]
If you're looking for a 5.11.1 version this "works": https://github.com/unitycontainer/microsoft-dependency-injection/pull/58

You should be able to pack into a nupkg and publish to your own private nuget server/local package directory.

Cheers,
P.

**Comment by ENikS** [comment]
Released 5.11.0 compatible with asp .net  2.2
